scripts/scil_compute_local_tracking.py

```diff
@@ -106,6 +106,11 @@ def _build_arg_parser():
         help='Random number generator seed.')
     add_overwrite_arg(out_g)
 
+    out_g.add_argument(
+        '--save_seeds', action='store_true',
+        help='If set, save the seeds of the streamlines alongside the '
+             'streamlines themselves')
```

Comment:
Not sure i like this expression. "alongside the" 
Why not say something like save the seeds in the data_per_point of the trk of the streamlines
or something like that